src/main/java/com/ernie/TicketApp/repository/UserDAO.java

```diff
@@ -93,4 +95,46 @@ public List<User> getAllUsers() {
         return allUsers;
     }
 
+    @Transactional
+    public void updateUserAddTicket(UUID id) {
+        User user = null;
+        Ticket newTicket = new Ticket();
+        newTicket.setTicketType(TicketType.MONTH);
+        String SQLStatementGetUser = "SELECT * FROM user_info WHERE id = ?";
+        try (PreparedStatement preparedStatement = dataSource.getConnection().prepareStatement(SQLStatementGetUser)) {
+            preparedStatement.setString(1, id.toString());
+            ResultSet resultSet = preparedStatement.executeQuery();
+            if (resultSet.next()) {
+                String name = resultSet.getString("name");
+                user = new User(name);
+                user.setId(UUID.fromString(resultSet.getString("id")));
+                user.setCreationDateTime(LocalDateTime.parse(resultSet.getString("creation_date")));
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
```

Comment:
this logic is not related to ` update User and create ticket ` task. 
1. move on to service class:
```java
        Ticket newTicket = new Ticket();
        newTicket.setTicketType(TicketType.MONTH);
        User user = userDAO.getUserById(...); // retrieve user
        userDAO.updateUserAddTicket(user, ....); //  update User and create ticket
```